hikari/api/rest.py

```diff
@@ -3275,6 +3275,9 @@ async def add_user_to_guild(
             Requires the `MANAGE_NICKNAMES` permission on the guild.
         nick : hikari.undefined.UndefinedOr[builtins.str]
             Deprecated alias for `nickname`.
+
+            .. deprecated:: 2.0.0.dev106
+                Use `nickname` instead.
         roles : hikari.undefined.UndefinedOr[hikari.snowflakes.SnowflakeishSequence[hikari.guilds.PartialRole]]
             If provided, the roles to add to the user when he joins the guild.
```

Comment:
you need to document the new restrictions around using these endpoints

Reply:
What restrictions?

Reply:
oh put that on the wrong line
but for the PUT permissions endpoint you need to document that its not a bot endpoint anymore and can must be used through the oauth2 flow with the perms scope.